### ISSUE TITLE
How to Effectively Share|Reuse Components

### ISSUE THREAD

**Issue by faburu** [react-cosmos/react-cosmos#651]
### What's up?

I am clueless on how to share components among projects.

### Mkay, tell me more...

So here's the dilly.
I am now acting like a good software person and developing my components in isolation.
And then it falls apart.
I copy and paste the components from Cosmos into my project.. and then into my other project.. tweak the component a little, copy it b. .... .. .. and then I sigh heavily.

I have tried several methods of sharing components among my projects and each rabbit hole is getting deeper. Currently, I am trying to use npm and Sinopia to manage the components, but get some webpack/babel error.
```js
Module parse failed: Unexpected token (33:11)
You may need an appropriate loader to handle this file type.
```

Now I need expert advice;
What is the proper workflow for creating my shiny components in Cosmos and importing them into other projects?

**Comment by skidding** [comment]
Good question. But I'm afraid it goes outside Cosmos' scope, at least currently.

Cosmos for ensures components can be reused in the same project, and is mostly concerned with run time dependencies. What you're refering to is more related to build time dependencies. That is your Babel presets & plugins, your webpack config, loaders, plugins, etc. 

One answer could be: Use create-react-app, or Next.js, or some other opinionated framework in all your projects. Then you should be able to copy paste components.

@ComputerCarl does this make sense?

**Comment by ComputerCarl** [comment]
Copying and pasting was the thing driving me to absolute madness.

I did find NWB and tried to use that, so I could at least build and import, to create a one-way workflow, but then I couldn't use it with React-Cosmos.

I found issue [#635](https://github.com/react-cosmos/react-cosmos/issues/635) which discusses the conflicting webpack configurations, but I don't understand the solution offered. Should I open another issue, or zombie post there?
